src/lib/wicked/files.test.js

```diff
@@ -173,13 +218,13 @@ describe('SysconfigFile', () => {
 
         it('returns the value for the given key', async () => {
             await file.read();
-            expect(file.get('NAME')).toBeUndefined();
-            expect(file.get('BOOTPROTO')).toEqual('dhcp');
-            expect(file.get('STARTMODE')).toBeUndefined();
+            expect(file.getKey('NAME')).toBeUndefined();
+            expect(file.getKey('BOOTPROTO')).toEqual('dhcp');
+            expect(file.getKey('STARTMODE')).toBeUndefined();
```

Comment:
I miss around here a test for the #getKey with a default value `file.getKey('STARTMODE', 'boot')`

Reply:
Good point. I have addded a test for that. Thanks!